library/blivet.py

```diff
@@ -1265,18 +1289,114 @@ def _manage_encryption(self, members):
 
         return managed_members
 
+    def _manage_thin_pools(self, pool_device):
+
+        def get_vol_size(volume):
+            if isinstance(volume, str) and '%' in volume:
+                try:
+                    percentage = int(volume[:-1].strip())
+                except ValueError:
+                    raise BlivetAnsibleError("invalid percentage '%s' size specified for volume in pool '%s'" % (pool_device.name))
```

Comment:
Error here - `TypeError: not enough arguments for format string` - you are missing the volume size str - so
```
                     raise BlivetAnsibleError("invalid percentage '%s' size specified for volume in pool '%s'" % (vol_size, pool_device.name))
```